lang/es_ES.json

```
@@ -0,0 +1,347 @@
+    "We will protect personal information by reasonable security safeguards against loss or theft, as well as unauthorized access, disclosure, copying, use or modification.": "Protegeremos la información personal mediante medidas de seguridad razonables contra pérdidas o robo, así como acceso, divulgación, copia, uso o modificación no autorizados.",
+    "We maintain API usage logs for a period deemed appropriate to ensure optimal service.": "Mantenemos registros de uso de la API por un período considerado apropiado para garantizar un servicio óptimo.",
+    "Information about cookies": "Información sobre cookies",
+    "You can prevent storage of cookies by appropriately setting your browser software; in this case, however, please note that you might not be able to fully use all functions offered by this website.": "Puede evitar el almacenamiento de cookies configurando adecuadamente su navegador. Sin embargo, en este caso, tenga en cuenta que es posible que no pueda utilizar completamente todas las funciones ofrecidas por este sitio web.",
+    "Reset Game Progress": "Restablecer progreso del juego",
+    "Reset Progress": "Restablecer progreso",
+    "Game": "Juego",
+    "Achievement": "Logro",
+    "Select an achievement": "Selecciona un logro",
+    "All won achievements for this game": "Todos los logros desbloqueados para este juego",
+    ":gameTitle (:consoleName) (:numAwarded / :numPossible won)": ":gameTitle (:consoleName) (:numAwarded / :numPossible desbloqueados)",
+    ":achievementTitle (:achievementPoints points)": ":achievementTitle (:achievementPoints puntos)",
+    "(Hardcore)": "(Hardcore)",
+    "Current Email Address": "Dirección de correo electrónico actual",
+    "New Email Address": "Nueva dirección de correo electrónico",
+    "Change Email": "Cambiar correo electrónico",
+    "Change Password": "Cambiar contraseña",
+    "Current Password": "Contraseña actual",
+    "enter your current password here...": "introduce aquí tu contraseña actual...",
+    "New Password": "Nueva contraseña",
+    "enter a new password here...": "introduce aquí una nueva contraseña...",
+    "Confirm Password": "Confirmar contraseña",
+    "confirm your new password here...": "confirma aquí tu nueva contraseña...",
+    "enter your new email address here...": "introduce aquí tu nueva dirección de correo electrónico...",
+    "Confirm New Email Address": "Confirmar nueva dirección de correo electrónico",
+    "confirm your new email address here...": "confirma aquí tu nueva dirección de correo electrónico...",
+    "Delete Account": "Borrar cuenta",
+    "Keys": "Claves",
+    "Notification type": "Tipo de notificación",
+    "Email notifications": "Notificaciones por correo electrónico",
+    "Site notifications": "Notificaciones en el sitio",
+    "Comments on my activity": "Comentarios en mi actividad",
+    "Comments on an achievement I created": "Comentarios en un logro ",
```

I think it should be:

```suggestion
    "Comments on an achievement I created": "Comentarios en un logro creado por mí",
```